spec/views/content_items/show.html.erb_spec.rb

```diff
@@ -60,6 +60,14 @@
     expect(rendered).to have_selector('td + td', text: '2 months ago')
   end
 
+  it 'renders a link to FeedEx' do
+    content_item.base_path = '/the-base-path'
+    feedex_link = "https://support.publishing.service.gov.uk/anonymous_feedback?path=/the-base-path"
+    render
+
+    expect(rendered).to have_link('View anonymous feedback on FeedEx', href: feedex_link)
```

Comment:
As per [Trello card](https://trello.com/c/zbEq1LBc/183-5-show-content-feedback-number-of-support-contacts-made-for-a-content-item) it seems that we need named and anonymous feedback?